btcwrapper/interface.go

```diff
@@ -1,26 +0,0 @@
-package btcwrapper
```

Comment:
I'm not sure but relayer can support btcd and bitcoind. Maybe this is the reason why we have btcwapper here.

Reply:
Got it!